neptune/alpha/sync.py

```diff
@@ -182,34 +185,38 @@ def list_experiments(path: Path, synced_experiments: Sequence[Experiment],
 
     if not unsynced_experiments:
         click.echo()
-        click.echo('There are no unsynchronized experiments in {}'.format(path))
+        click.echo('There are no unsynchronized experiments in {}'.format(base_path))
 
     if not synced_experiments:
         click.echo()
-        click.echo('There are no synchronized experiments in {}'.format(path))
+        click.echo('There are no synchronized experiments in {}'.format(base_path))
 
     click.echo()
     click.echo('Please run with the `neptune sync --help` to see example commands.')
 
 
-def synchronization_status(path: Path) -> None:
-    synced_experiments, unsynced_experiments = partition_experiments(path)
-    offline_experiments_ids = get_offline_experiments_ids(path)
-    list_experiments(path, synced_experiments, unsynced_experiments, offline_experiments_ids)
+def synchronization_status(base_path: Path) -> None:
+    synced_experiments, unsynced_experiments = partition_experiments(base_path)
+    offline_experiments_ids = get_offline_experiments_ids(base_path)
+    list_experiments(base_path, synced_experiments, unsynced_experiments, offline_experiments_ids)
 
 
 #######################################################################################################################
 # Experiment synchronization
 #######################################################################################################################
 
 
-def sync_experiment(path: Path, qualified_experiment_name: str) -> None:
-    experiment_uuid = uuid.UUID(path.name)
+def sync_experiment(experiment_path: Path, qualified_experiment_name: str) -> None:
+    experiment_uuid = uuid.UUID(experiment_path.name)
     click.echo('Synchronising {}'.format(qualified_experiment_name))
+    for execution_path in experiment_path.iterdir():
+        sync_execution(execution_path, experiment_uuid, qualified_experiment_name)
+
 
-    disk_queue = DiskQueue(str(path), OPERATIONS_DISK_QUEUE_PREFIX,
+def sync_execution(execution_path: Path, experiment_uuid: uuid.UUID, qualified_experiment_name: str) -> None:
+    disk_queue = DiskQueue(str(execution_path), OPERATIONS_DISK_QUEUE_PREFIX,
                            VersionedOperation.to_dict, VersionedOperation.from_dict)
-    sync_offset_file = SyncOffsetFile(path)
+    sync_offset_file = SyncOffsetFile(execution_path)
     sync_offset = sync_offset_file.read() or 0
 
     while True:
```

Comment:
Printing "Synchronization of experiment..." below is no longer accurate, need to fix.